added migration to delete contact migration

Cut3cumber opened the pull request.

Cut3cumber committed:
added migration to delete contact migration

Cut3cumber committed:
changelog

tuerlefl commented on CHANGELOG.md:
```diff
@@ -32,6 +32,7 @@ general bugfixes
 
 #### DATA MIGRATIONS
 - Added custom link option for pagination button on content plugins [#118](https://github.com/allink/allink-core/pull/118)
+- Removed contact migration [#125](https://github.com/allink/allink-core/pull/125)
```
what happens, if the config app is overridden on a project? We should at least add a comment here.

Cut3cumber committed:
updated changelog

tuerlefl reviewed (approved):
*please resolve conflicts

Cut3cumber committed:
Cleaned up migrations
Merge branch 'v2.8.0' into amal/fix/delete-contact-migration

# Conflicts:
#	CHANGELOG.md

Cut3cumber committed:
Merge branch 'v2.8.0' into amal/fix/delete-contact-migration

# Conflicts:
#	CHANGELOG.md
